File: DayTwelve/Java/GcdOfNumber.java
public class GcdOfNumber{
    public static int commonFactor(int numberOne, int numberTwo){

        int gcd = 0;

        for (int count = 1; count <= numberOne && count <= numberTwo; count++) {

            if (numberOne % count == 0 && numberTwo % count == 0) {
                gcd = count;
            }
        }

        return gcd;
    }

    public static void main(String[] args) {

        int numberOne = 12;
        int numberTwo = 18;

        int result = commonFactor(numberOne, numberTwo);

        System.out.println(result);
    }
}

File: DayTwelve/Python/gcd_of_number.py
def gcd(number_one, number_two):

    common_number = 1
    for number in range(1, number_one +1):

        if number_one % number == 0 and number_two % number == 0:

            common_number = number

    return common_number

number_one = 12

number_two = 18

result = gcd(number_one, number_two)

print(result)
            
    
